### PR TITLE
#5 added rust base project

### PR DESCRIPTION
**Pull request by muhammadmnorouzi**
[#5 ] adding rust language support for "FileManager" according to CSharp project and rust api development guidlines in [here](https://rust-lang.github.io/api-guidelines/about.html)

### CODE REVIEW

File: src/Rust/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[package]
+name = "easy_microservice"
```
Comment (Ali-YousefiTelori) [inline src/Rust/Cargo.toml:2]: Is the easy_microservice packages name?

**Review by Ali-YousefiTelori — CHANGES_REQUESTED** [review]
I need to know real package's name.
How it works in rust?

File: src/Rust/Cargo.toml (hunk 1 above)
Comment (muhammadmnorouzi) [reply src/Rust/Cargo.toml:2]: yes, it will be used like this

`
use easy_microservice::file_manager;
`

Comment (Ali-YousefiTelori) [reply src/Rust/Cargo.toml:2]: Ok, do we need to change it for now?

Comment (muhammadmnorouzi) [reply src/Rust/Cargo.toml:2]: There is no need to change it.
Also in crates.io we see no easy_microservice package , that means we can push it as a crate and reserve it for future uses.

**Review by Ali-YousefiTelori — APPROVED** [review]
Thanks for your contributions @muhammadmnorouzi.

**Review by mahdidelzendeh72 — APPROVED** [review]

**Pull request merged**